components/search/SearchSuggestion.vue

```diff
@@ -37,9 +37,8 @@
           :to="{
             path: `/${urlPrefix}/explore/collectibles`,
             query: { ...$route.query },
-          }"
-          @click.native="$emit('close')">
-          <div :class="loadMoreItemClassName">
+          }">
+          <div :class="loadMoreItemClassName" @click="seeAllCollection">
```

Comment:
There is another `see all` button for the gallery items. Pls also add the click handler to it and rename `seeAllCollection` to `seeAllButtonHandler`

Reply:
@click event on `<div>` ?